packages/loader/container-loader/src/container.ts

```diff
@@ -1397,7 +1413,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     }
 
     private processRemoteMessage(message: ISequencedDocumentMessage): IProcessMessageResult {
-        const local = this._clientId === message.clientId;
+        const local = this.clientIdHistory.includes(message.clientId);
+
+        // Report if we're getting messages from a clientId older than the most recent
+        if (local && this._clientId !== message.clientId) {
+            this.logger.sendErrorEvent({ eventName: "matchedOldClientIdInRemoteMessage" });
```

Comment:
Would it be useful to include any information about the matching clientId in the telemetry event? If we are expecting this to be infrequent/never, adding more relevant information might be useful.

Reply:
i could add the index of the matched clientid?  that could tell us if there are messages from multiple older clientids in flight simultaneously, but not sure if there's anything else relevant that's available here.

Reply:
I was thinking the clientId could be added. Not sure how useful it would be, but maybe sequenceNumber or other info from the op? Yeah it seems like it will be hard to know what we will need.